lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -936,363 +923,6 @@ def model_prm_baseline_system_groups(model, custom)
-      final_groups << comp_room_psz_group
-    end
-    unless dist_comp_room_svav_zones.empty?
-      dist_comp_room_svav_group = {}
-      dist_comp_room_svav_group['occ'] = hvac_building_type
-      dist_comp_room_svav_group['fuel'] = 'districtcooling'
-      dist_comp_room_svav_group['zone_group_type'] = 'computer_zones'
-      area_m2 = 0
-      dist_comp_room_svav_zones.each do |zone|
-        area_m2 += zone.floorArea * zone.multiplier
-      end
-      area_ft2 = OpenStudio.convert(area_m2, 'm^2', 'ft^2').get
-      dist_comp_room_svav_group['group_area_ft2'] = area_ft2
-      dist_comp_room_svav_group['building_area_type_ft2'] = total_area_ft2
-      dist_comp_room_svav_group['zones'] = dist_comp_room_svav_zones
-      # store info if any zone has district, fuel, or electric heating
-      dist_comp_room_svav_group['fuel'] = get_group_heat_types(model, dist_comp_room_svav_zones)
-      final_groups << dist_comp_room_svav_group
-    end
-    unless dist_comp_room_psz_zones.empty?
-      dist_comp_room_psz_group = {}
-      dist_comp_room_psz_group['occ'] = hvac_building_type
-      dist_comp_room_psz_group['fuel'] = 'districtcooling'
-      dist_comp_room_psz_group['zone_group_type'] = 'computer_zones'
-      area_m2 = 0
-      dist_comp_room_psz_zones.each do |zone|
-      end
-      area_ft2 = OpenStudio.convert(area_m2, 'm^2', 'ft^2').get
-      dist_comp_room_psz_group['group_area_ft2'] = area_ft2
-      dist_comp_room_psz_group['building_area_type_ft2'] = total_area_ft2
-      dist_comp_room_psz_group['zones'] = dist_comp_room_psz_zones
-      # store info if any zone has district, fuel, or electric heating
-      dist_comp_room_psz_group['fuel'] = get_group_heat_types(model, dist_comp_room_psz_zones)
-      final_groups << dist_comp_room_psz_group
-    end
-
-    ngrps = final_groups.count
-    # Determine the number of stories spanned by each group and report out info.
-    final_groups.each do |group|
-      # Determine the number of stories this group spans
-      num_stories = model_num_stories_spanned(model, group['zones'])
-      group['stories'] = num_stories
-      # Report out the final grouping
-      OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', "Final system type group: occ = #{group['occ']}, fuel = #{group['fuel']}, area = #{group['group_area_ft2'].round} ft2, num stories = #{group['stories']}, zones:")
-      group['zones'].sort.each_slice(5) do |zone_list|
-        zone_names = []
-        zone_list.each do |zone|
-          zone_names << zone.name.get.to_s
-        end
-        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', "--- #{zone_names.join(', ')}")
-      end
-    end
-
-    return final_groups
-  end
-
   # Before deleting proposed HVAC components, determine for each zone if it has district heating
   # @return [Hash] of boolean with zone name as key
   def get_district_heating_zones(model)
```

Comment:
To be consistent with other methods, the name should probably be `model_get_district_heating_zones` since it's under `Standards.Model.rb`